### DISCUSSION TITLE
Host status not green if online

### DISCUSSION THREAD

**Discussion by akadawa** [seriousm4x/UpSnap#478] [Help wanted] [answered]
![image](https://github.com/seriousm4x/UpSnap/assets/32989761/d89ae6e5-43ce-4eb1-9fb3-62859e765d3c)

if the host is online, the status is not green. how can i fix it?

**Comment by seriousm4x** [comment]
Is it a windows computer? You might have to allow pings in the firewall.

Is ping from the command line working?

**Reply by akadawa** [reply]
Hey, i cant pring from the Docker Host. Clients like Android or Windows can Ping the client

**Reply by akadawa** [reply, marked as answer]
i added a fw rule for icmp to the windows host.

thank you
